core/controllers/reader.py

```diff
@@ -497,7 +505,7 @@ class RatingHandler(base.BaseHandler):
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
-    @require_playable
+    @acl_decorators.can_view_exploration_stats
```

Comment:
I think this probably goes under "can_play_exp", actually. Even if someone can't see the full stats they should be able to see at least the overall and their own ratings of the exploration.